ADR/ADR023-paramaterise-common-terraform-roots.md

```diff
@@ -0,0 +1,88 @@
+# ADR023: Paramaterise Common Terraform Roots
```

Comment:
ADR23 has already been taken along with ADR24 which is in another PR that Laurence has created

Reply:
thanks Al, I've updated the number